revisions.html.md.erb

```diff
@@ -63,43 +63,66 @@ This section describes how to use CAPI endpoints for viewing revisions.
 
 To list revisions for an app:
 
-1. Retrieve the GUID of the app by running:
-
+1. Using the `revisions` command:
+	 
 	```
-	cf app APP-NAME --guid
```

Comment:
I think these indents may be too large -- they may cause display problems.
The standard indents are 2 spaces or 4 spaces. Larger than that causes trouble.